### DISCUSSION TITLE
QField Cloud : problème de synchronisation

### DISCUSSION THREAD

**Discussion by geguill** [opengisch/QField#4671] [Q&A]
Bonjour,
Je suis arrivé à créer un projet dans le cloud depuis QGIS.
J'ai réussi à le télécharger sur mon smartphone, et j'ai collecté des données de terrain. J'ai poussé les modifications dans le cloud sans problème.
J'ai vérifié sur le site qfield.cloud : OK.
De retour au bureau devant mon projet QGIS (QFieldCloud), la synchronisation du projet cloud ne se fait pas. J'ai le message suivant : "Le projet cloud stocké localement est déjà synchronisé avec QField Cloud, aucune action n'est requise."
 Bizarre ! ?
Que faut-il faire ?
Je joins mon dossier présent dans C:\Users\xxxx\QField\cloud
D'avance merci
Gérard
[chaource.zip](https://github.com/opengisch/QField/files/13060411/chaource.zip)

**Comment by dabpc** [comment]
Are you correctly "pushing" your data to QFieldCloud, rather than "synching" on your QField device?

**Comment by INTIAPRG** [comment]
geguill I have the same problem! Have you solved it?

**Comment by ChimmyChurry** [comment]
I also have the problem that qfield + qfieldcloud sync without problems but not in qgis via qfieldsync.
One solution is simply to delete all local files on PC. Then the (clean) sync shows all changes. 
But if there are already local geopackages, these are not getting updated (no matter with which qfield project settings).
Maybe something is not working with the file access or the version synchronization.

Windows 11
Qfield Sync 4.6.0
Qfield Cloud Community - Free
QGIS 3.28-9

**Reply by dabpc** [reply]
Sync from QFieldCloud to QGIS is working OK with:
Windows 10 Pro
Qfield Sync 4.6.0
Qfield Cloud Community - Free
QGIS 3.22-14

**Comment by BoswachterMarc** [comment]
_Je joins mon dossier présent dans C:\Users\xxxx\QField\cloud D'avance merci Gérard [chaource.zip](https://github.com/opengisch/QField/files/13060411/chaource.zip)_

It looks like your zip file is missing both data/gpkg files. I assume they are not in the same folder as you're project file/qgs? Perhaps that's part of the problem.
When I unzip, there is only the chaource_cloud.qgs file.

![image](https://github.com/opengisch/QField/assets/57868975/ac825164-a118-47f4-8f43-e1920a92f99d)

![image](https://github.com/opengisch/QField/assets/57868975/c1f80f99-f1ca-4077-9ef5-fea6316dbf46)

**Reply by geguill** [reply]
Bonjour,
le fichier .zip est complet.
